### PR TITLE
Safer math command

### PR DESCRIPTION
**Pull request by BlueSlimee**
The current math command has a serious problem: math operations are executed in the same event loop as the bot. The problem? Heavy math operations (such as `:Infinity` or `:42000000`) will crash the bot. This PR sandboxes all math operations so everything is isolated and safe. Currently, the execution timeout is 10 seconds, but I think it can be brought down to 5 or something.

### CODE REVIEW

**Comment by metehus** [comment]
#895

**Comment by metehus** [comment]
image

**Review by metehus — DISMISSED** [review]
Sounds smart and good

**Comment by metehus** [comment]
ok something broke

**Review by almeidx — CHANGES_REQUESTED** [review]

File: src/commands/utility/math.js [hunk 1]
```diff
@@ -27,8 +16,8 @@ module.exports = class Math extends Command {
   async run ({ t, author, channel }, expression) {
     const embed = new SwitchbladeEmbed(author)
     try {
-      const result = math.eval(expression)
-      embed.setTitle(t('commands:math.result', { result }))
+      const data = await this.client.apis.mathematics.calculate(expression).then(json => JSON.parse(json))
+      embed.setTitle(t('commands:math.result', { result: data._data || data }))
     } catch (error) {
       this.client.log(`Failed math calculation ${expression}\nError: ${error.stack}`, { tags: [this.name] })
```
Comment (almeidx) [inline src/commands/utility/math.js:22]: This should be removed imo

**Review by almeidx — APPROVED** [review]
Besides the above, LGTM

File: src/commands/utility/math.js (hunk 1 above)
Comment (almeidx) [inline src/commands/utility/math.js:22]:
```suggestion
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by almeidx — APPROVED** [review]
LGTM

**Review by Doges — APPROVED** [review]
LGTM

**Pull request merged**